package.json

```diff
@@ -9,7 +9,6 @@
   },
   "author": "Jozef Hartinger",
   "dependencies": {
-    "express-http-context": "^1.2.4",
```

Comment:
@rexwhite It looks like it removed this line because it was a duplicate but I can undo it if you want? It doesn't look like it changed anything in package-lock

Reply:
Weird.  Yeah, put express-http-context back.

Reply:
Right.. Removed it 👍